source/client/service_impl.h

```diff
@@ -31,14 +31,22 @@ class ServiceImpl final : public nighthawk::client::NighthawkService::Service,
 
 private:
   void handleExecutionRequest(const nighthawk::client::ExecutionRequest& request);
-  void writeResponseAndFinish(const nighthawk::client::ExecutionResponse& response);
+  void writeResponse(const nighthawk::client::ExecutionResponse& response);
   ::grpc::Status finishGrpcStream(const bool success, absl::string_view description = "");
 
   Envoy::Event::RealTimeSystem time_system_; // NO_CHECK_FORMAT(real_time)
   Envoy::Thread::MutexBasicLockable log_lock_;
-  ::grpc::ServerReaderWriter<::nighthawk::client::ExecutionResponse,
-                             ::nighthawk::client::ExecutionRequest>* stream_;
+  static ::grpc::ServerReaderWriter<::nighthawk::client::ExecutionResponse,
```

Comment:
It's static so we can assert when multiple service instances are created because of multiple clients connecting; we assert when we do so.. any suggestions?